mono_repo/test/shared.dart

```diff
@@ -58,8 +58,15 @@ dart:
 
 dart_task:
  - test: --platform dartium
```

Comment:
While we're here, remove all references to dartium?

Reply:
Eh, it's still a valid configuration. Doesn't affect anything at runtime. Just verifying that we parse things correctly.

This could be `--platform silly_bunny`

Reply:
Yeah, more a comment that we should completely purge the dartium brand from our collective memory.

Just use `--platform chrome` instead?

Reply:
Gah...fine